ClassAttendance/src/WebUI/Controllers/AccountController.cs

```diff
@@ -0,0 +1,111 @@
+﻿using BLL.Exceptions;
+using Microsoft.AspNetCore.Authorization;
+using Microsoft.AspNetCore.Mvc;
+using Microsoft.Extensions.Logging;
+using System;
+using System.Threading.Tasks;
+using WebUI.Facades;
+using WebUI.Models;
+using WebUI.Models.Account;
+
+namespace WebUI.Controllers
+{
+    public class AccountController : Controller
+    {
+        private ILogger<AccountController> _logger;
+        private AccountFacade _accountFacade;
+
+        public AccountController(ILogger<AccountController> logger, AccountFacade accountFacade)
+        {
+            _logger = logger;
+            _accountFacade = accountFacade;
+        }
+
+        public Uri Referer => new Uri(Request.Headers["Referer"].ToString());
```

Comment:
Maybe an extension method for request object would be better?

Reply:
Done